src/community/features-templating/features-templating-ogcapi/src/test/java/org/geoserver/featurestemplating/response/JSONLDGetComplexFeaturesResponseAPITest.java

```diff
@@ -141,24 +141,6 @@ public void testJsonLdQueryEnvSubstitutionOnXpath() throws Exception {
         }
     }
 
-    @Test
```

Comment:
Ouch?

Reply:
The check on the backward mapping assumed that the template path starts with "features" due its origin a json templating module. In order to allow other output format as gml and then html the best thing seemed to me to remove it and log the path if not resolvable against a template. Then it will be passed down to the store and exception will be eventually thrown if does not get validated against the schema.